### PR TITLE
fix(aap): fix config loader issue on backstage v1.17

### PR DESCRIPTION
**Pull request by invincibleJai**
Fixes: https://github.com/janus-idp/backstage-plugins/issues/658

### CODE REVIEW

**Review by tumido — APPROVED** [review]
/lgtm
/approve

File: plugins/aap-backend/config.d.ts [hunk 1]
```diff
@@ -1,13 +1,4 @@
...
-
-interface AapConfig {
-  baseUrl: string;
-  /** @visibility secret */
-  authorization: string;
-  system?: string;
-  owner?: string;
-  schedule?: TaskScheduleDefinitionConfig;
-}
+import { AapConfig } from './src/types';
```
Comment (tumido) [inline plugins/aap-backend/config.d.ts:1]: Hold on a minute... I'm not sure this works as intended... I'll need to take a look into the build process.

Comment (tumido) [reply plugins/aap-backend/config.d.ts:1]: How does this get references when there's no `src` folder in the built NPM package and `config.d.ts` is not transpiled into JS? It seems this reference doesn't exist/never gets imported.

Comment (invincibleJai) [reply plugins/aap-backend/config.d.ts]: Have updated it have inline

**Pull request closed without merging**